frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -20,9 +19,26 @@
       "notes": "",
       "versus": ""
     },
+    "middleware-json": {
+      "setup_file": "setup-json",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "None",
+      "framework": "aspnetcore",
+      "language": "C#",
+      "orm": "Raw",
+      "platform": "NET",
```

Comment:
As always, this stuff is nuanced :smile:

.NET is a platform in the same way Java is. However I see in many other places that platform is used to indicate an application stack, independent of the language or runtime, e.g. Netty is used as a platform, although it's written in and runs on Java, similarly "Servlet" is a platform.

Also, the official description of platform seems to indicate it's the thing that provides the HTTP primitives the application is built on, rather than the underlying runtime or execution environment:
> "platform" is the low-level software or API used to host web applications for the framework; the platform provides an implementation of the HTTP fundamentals.

In this vein, it could be viewed that the platform in our case is "ASP.NET Core", vs. "ASP.NET" or some other .NET-based web framework, e.g. Nancy, as ASP.NET Core defines the primitives on which the web application is written. Another .NET web application platform, like Nancy, has its own primitives.

"NETCoreAOT" is a future pivot where the application has been run through the .NET native compiler (AOT == Ahead of Time vs. JIT == Just in Time, in .NET parlance) and hence has a different .NET runtime (CoreRT) embedded in it, rather than running on the CLR or CoreCLR with JIT compilation, etc.

All that said, does it sound more like perhaps we should be classifying these along these lines:
- language = C#
- platform = aspnetcore for our tests. If Nancy submits tests, they'd be "nancy"
- classification = "Platform" for when we use Kestrel (our server) directly without middleware at all (we don't have a test for this right now), "Micro" for raw middleware or routing, "FullStack" for MVC controllers
- front-end-server = None in our case, as the framework includes a server in-process (Kestrel)
- flavor = CoreCLR or CLR  or CoreRT or Mono

?